src/infra/internalDatabase/model/marketplaceInstalledItem.go

```diff
@@ -12,6 +12,7 @@ import (
 type MarketplaceInstalledItem struct {
 	ID               uint `gorm:"primarykey"`
 	Name             string
+	Hostname         string
```

Comment:
Acho que seria uma boa ideia ter o VirtualHostId e VirtualHostHostname ao invés de apenas "Hostname". Assim fica mais óbvio do que se trata e caso seja preciso ter o Id do Hostname, tá ready.

Reply:
Como combinado essa request change foi movida para o to-do da tarefa de refact do vhost. A ideia agora vai ser relacionar a model do vhost com essa model assim que toda sua estrutura for convertida para SQLite.